int32 predefined_cmd
float32 wheel_velocity
float32 crab_angle
bool use_crab_angle
bool force_cmd
